bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -3527,7 +3542,27 @@
         <comment>New roles and privileges for bahmni</comment>
         <sqlFile path="V1_96_NewRolesAndPrivileges.sql"/>
     </changeSet>
-    <changeSet id="map-users-to-new-roles" author="Jaswanth, Ravindra">
+    <changeSet id="bahmni-core-201906171037" author="Samuel34">
+    	<preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                select count(*) from users where system_id='superman';
+            </sqlCheck>
+        </preConditions>
+        <comment>Adding superman user</comment>
```

Comment:
Details of this changeset : https://bahmni.slack.com/archives/C03LWT96L/p1560755769006900

Reply:
:heavy_check_mark: